conda/base/context.py

```diff
@@ -487,6 +496,19 @@ def plugin_manager(self):
 
         return get_plugin_manager()
 
+    @property
+    def no_plugins(self):
+        """
+        This property is used to determine if plugins should be loaded or not.  It is set by
+        the ``--no-plugins`` command line flag.  It can also be set by the ``NO_PLUGINS``
+        environment variable.  If neither of those are set, then the default value is used.
+        """
+        return (
+            self._argparse_args
+            and "no_plugins" in self._argparse_args
+            and self._argparse_args["no_plugins"] is not None
+        ) or os.getenv("NO_PLUGINS") is not None
```

Comment:
This is also missing the fallback to `self._no_plugins` if `self._argparse_args` doesn't define `--no-plugins`.

Reply:
```suggestion
        return (
            self._argparse_args
            and "no_plugins" in self._argparse_args
            and self._argparse_args["no_plugins"] is not None
        ) or self._no_plugins
```

Would that suffice?